core/src/main/java/org/geysermc/geyser/network/MinecraftProtocol.java

```diff
@@ -61,7 +61,10 @@ public final class MinecraftProtocol {
         SUPPORTED_BEDROCK_CODECS.add(Bedrock_v527.V527_CODEC.toBuilder()
                 .minecraftVersion("1.19.0/1.19.2")
                 .build());
-        SUPPORTED_BEDROCK_CODECS.add(DEFAULT_BEDROCK_CODEC);
+
+        SUPPORTED_BEDROCK_CODECS.add(DEFAULT_BEDROCK_CODEC.toBuilder()
```

Comment:
These changes are completely unnecessary as the Minecraft version of 1.19.10 is already "1.19.10".